DashAISoftware/DashAI
# Remove styled-components

# Summary


Remove [styled-components](https://styled-components.com/docs) and [mui package to use styled-components as style engine](https://mui.com/material-ui/guides/styled-engine/). Refactor certain components to eliminate dependencies on styled-components.

## Type of change


- Refactoring.

## Changes


- Uninstall `styled-components` and `@mui/styled-engine-sc`
- Delete `styles/components` folder and all its content.
- Move `InputStyles.jsx` to `ConfigurableObject/Inputs/` folder
- Delete `globalComponents.js` 
- Remove styled-components `GlobalStyles`
- Delete the original `theme.js` and rename `muiGlobalStyles.js` to `theme.js`  
- Refactor some components to remove the remaining use of styled-components

## How to Test


1. Go to `DashAI/DashAI/front` and run `yarn json-server`
2. in another terminal in the same folder of step 1 run `yarn start`
3. The app should work normally

## Notes


* This PR depends on #86 

* This PR has a lot of files changed, but the majority of them involve the removal of files responsible for handling styling (11 files) or modifications to imports due to component relocations/renaming (6 files).
